### DISCUSSION TITLE
Is it good idea, first: 3D scan environment (mapping), and Next: Just localizations ? (not Simultaneous )

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1187] [General]
Hi

I know SLAM, means Simultaneous localization and mapping
but, what if I first make a good 3D scan of environment by human using Kinect like camera and save the file

and in operation time, just localize robot using Kinect+rtabmap+saved map ?

**Comment by matlabbe** [comment]
Yes, it is often the most common workflow for robot navigation:
1. Teleop (or send goals if navigation stack is up) the robot with rtabmap in SLAM mode (default) to create the map
2. After the map is created, restart rtabmap in [LOCALIZATION](http://wiki.ros.org/rtabmap_ros/Tutorials/HandHeldMapping#Localization_mode) mode (set parameter `Mem/IncrementalMemory` to `false` and remove `--delete_db_on_start` argument if previously set). Rtabmap will localize in previous map without adding new data.
